static/js/tabs.js

```diff
@@ -0,0 +1,126 @@
+/**
+  Updates the URL with the id of the target tab content
+  while preventing the page from jumping around by
+  manipulating the history object. Inspired by:
+  // https://gist.github.com/pimterry/260841c2104f27cadc954a29b9873b96#file-disable-link-jump-with-workaround-js
+  @param {HTMLElement} tab the triggered tab
+  @param {Array} tabs an array of sibling tabs
+*/
+function handleTabInteraction(tab, tabs) {
+  history.pushState({}, "", tab.href);
+  history.pushState({}, "", tab.href);
+  history.back();
+
+  setActiveTab(tabs);
+}
+
+/**
+  Attaches a number of events that each trigger
+  the reveal of the chosen tab content
+  @param {Array} tabs an array of tabs within a container
+*/
+function attachEvents(tabs) {
+  tabs.forEach(function (tab) {
```

Comment:
This code is not transpiled, so `forEach` will not work in older browsers. You should also convert the dom elements list to an array with `[].slice.call()` or similar.

Reply:
> You should also convert the dom elements list to an array with [].slice.call() or similar.

Yep, this already happens on [line 80](https://github.com/canonical-web-and-design/juju.is/pull/194/files#diff-b85bda63b95d30915f3f0e42806a0c674460ae7b8868fd028ca21020bda88b0aR80) (there are a couple of instances of `NodeList`s being cast to `Array`s)